packages/blockchain-wallet-v4-frontend/src/components/Form/AmountFieldInput/AmountFieldInput.stories.tsx

```diff
@@ -73,3 +74,15 @@ FiatOnly.args = {
   showCounter: false,
   showToggle: false
 }
+
+export const InitialValueZero = Template.bind({})
+InitialValueZero.args = {
+  amtError: false,
+  'data-e2e': 'fooo',
```

Comment:
add valid description

Reply:
I changed to `'data-e2e': 'amountField'`